VHbbAnalysis/Heppy/test/combined_cmssw.py

```diff
@@ -647,11 +647,26 @@ def initialize(**kwargs):
         setupAllVIDIdsInModule(process, eleid, setupVIDElectronSelection)
     process.OUT.outputCommands.append("keep *_electronMVAValueMapProducer_*_EX")
     process.OUT.outputCommands.append("keep *_egmGsfElectronIDs_*_EX")
+
+    
+
+    ########################################
+    # MET significance matrix
+    ########################################
     
+    from PhysicsTools.PatUtils.tools.runMETCorrectionsAndUncertainties import runMetCorAndUncFromMiniAOD
+    isData = not isMC
+    runMetCorAndUncFromMiniAOD(process, isData = isData)
+    process.OUT.outputCommands.append("keep *_slimmedMETs_*_EX")
+
+    processDumpFile = open('combined_cmssw.dump', 'w')
```

Comment:
Hi Andrea,

I have removed the python dump from PR #527.

Cheers,


Christian

On Aug 31, 2016, at 6:32 PM, arizzi wrote:

> In VHbbAnalysis/Heppy/test/combined_cmssw.py:
> 
> >      
> > +    from PhysicsTools.PatUtils.tools.runMETCorrectionsAndUncertainties import runMetCorAndUncFromMiniAOD
> > +    isData = not isMC
> > +    runMetCorAndUncFromMiniAOD(process, isData = isData)
> > +    process.OUT.outputCommands.append("keep *_slimmedMETs_*_EX")
> > +
> > +    processDumpFile = open('combined_cmssw.dump', 'w')
> 
> no, this is creating a dump file and printing it... why?
> 
> �
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>